google/resource-snippets/cloudkms-v1/test_alternatives.sh

```diff
@@ -0,0 +1,39 @@
+set -e
+
+# assumes that the following projects are created and kubectl context is set to a cluster in KRM_PROJECT
+TF_PROJECT_ID=[TF_PROJECT_ID]
+DM_PROJECT_ID=[DM_PROJECT_ID]
+
+gcloud auth application-default login
```

Comment:
One more thing. Don't we need the user to create Application Default Credentials after `gcloud config set project $DM_PROJECTID` *and* after `gcloud config set project $TF_PROJECTID` ? I'm not sure how this is working currently (maybe because your user account has owner access to both projects).

Reply:
Added a credential session in readme.